Code/Framework/AzCore/Tests/EventTests.cpp

```diff
@@ -257,6 +257,48 @@ namespace UnitTest
         EXPECT_FALSE(testEvent1.HasHandlerConnected());
         EXPECT_TRUE(testEvent2.HasHandlerConnected());
     }
+
+    TEST_F(EventTests, TestHandlerCopyConstructorOperator)
+    {
+        int32_t invokedCounter = 0;
+
+        AZ::Event<> testEvent;
+        AZ::Event<>::Handler testHandler([&invokedCounter]() { invokedCounter++; });
+
+        testHandler.Connect(testEvent);
```

Comment:
Suggestion: `EXPECT_FALSE(testHandler.IsConnected());` before this line to affirm that we're not connected until the call to `Connect` (i.e. `IsConnected()` doesn't just return `true` regardless of connection state).

Reply:
There are other tests checking the connection specifically. No need to test that here too, keeping this test more focus.